secure_redis/migrations/0001_initial.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
@GhalebKhaled I understand your reasoning, but I disagree. I think we shouldn't internally support migration like this, as it can be dangerous. We should only support keeping encrypted values in a given cache. Its possible to define 2 caches (one encrypted, one insecure) and have them point at the same database, so long as they have different prefixes, but I don't think that's a good idea to support.

If someone wants to migrate from plain text to encrypted values, its easy enough to (a) define the encrypted cache on a different database, and (b) write their own management command to migrate the data.

I don't think we should support this as internal functionality (though we can document a recommendation about how to do this in the README file.